tests/e2e/conftest.py

```diff
@@ -2,86 +2,64 @@
 
 import pytest
 import simplejson as json
-from django.template import (Context,
-                             Template)
 
-from treeherder.client.thclient import TreeherderJobCollection
 from treeherder.etl.jobs import store_job_data
 
 base_dir = os.path.dirname(__file__)
 
 
 @pytest.fixture
-def pending_jobs():
+def pending_job():
```

Comment:
These fixtures only return a single job, so I've adjusted their names to be more accurate